src/chttpd_db.erl

```diff
@@ -93,7 +93,7 @@ handle_changes_req1(#httpd{}=Req, Db) ->
             couch_stats:decrement_counter([couchdb, httpd, clients_requesting_changes])
         end;
     _ ->
-        Msg = <<"Supported `feed` types: normal, continuous, longpoll, eventsource">>,
+        Msg = <<"Supported `feed` types: normal, continuous/stream, longpoll, eventsource">>,
```

Comment:
I agree with @kxepal that `stream` should be listed like the other options.

I'm -0 on adding an alias for an existing feature (that is, I'd rather we didn't add more api without adding functionality, but I'm not blocking it).